R/pkg/NAMESPACE

```diff
@@ -403,6 +403,7 @@ export("as.DataFrame",
        "refreshTable",
        "setCheckpointDir",
        "setCurrentDatabase",
+       "setJobDescription",
```

Comment:
I think this should go to https://github.com/HyukjinKwon/spark/blob/c591e63d5937403bb462bdf76795af8e3f324aca/R/pkg/NAMESPACE#L78